### PR TITLE
Convert html to Markdown file (dev license)

### PR DESCRIPTION
**Pull request by Andre601**
Converts the html file used for the dev license to markdown.
Additionally were images used by this renamed to make more sense.

I hope nothing breaks here.

### CODE REVIEW

File: developer-license.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
```
Comment (Rexogamer) [inline developer-license.md:1]: What about creationDate?

File: developer-license.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+title: Developer Application License
+description: Information about Discord's Developer Application License which is used to publish games.
+published: true
+date: 2020-10-15T21:50:52.089Z
+tags: 
+---
+
+Discord's Developer Application License (commonly know as the **gamedev license**) is a license that you need to buy if you want to publish your game on Discord.
```
Comment (Rexogamer) [inline developer-license.md:9]: Slight typo I made:
```suggestion
Discord's Developer Application License (commonly known as the **gamedev license**) is a license that you need to buy if you want to publish your game on Discord.
```

**Review by Rexogamer — APPROVED** [review]

File: developer-license.md (hunk 1 above)
Comment (Andre601) [reply developer-license.md:1]: Not needed in any way. See the verified dev page for example.

Comment (Rexogamer) [reply developer-license.md:1]: Recently, when I've been updating older articles the creation date has been added automatically which might be a sign that it's good to have but oh well

File: developer-license.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+# News Channels
+News Channels allow you publish (also known as "crossposting") messages to any other server that follows this particular server.  
+You publish a message by clicking the "publish" icon next to the message (The megaphone icon). *This requires you to either be the author of the message or to have Manage Message Perms!*
+
+![news_channel.png](/uploads/developer-license/news_channel.png 'An example of a News Channel.')
+
+# Store Channels
+Store Channels are channels you can see game details at, they're similar to store pages but they're contained inside your server on a channel, these channels have a naming structure similar to voice channels which allows uppercase characters and spaces in their names. A store channel may look like this.
+
+![](/uploads/developer-license/store_channel.png 'An example of a Store Channel.')
```
Comment (Rexogamer) [inline developer-license.md:26]:
```suggestion
![store_channel.png](/uploads/developer-license/store_channel.png 'An example of a Store Channel.')
```

File: developer-license.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+The form for creating a store channel:
+
+![store_channel_form.png](/uploads/developer-license/store_channel_form.png)</figure>
```
Comment (Rexogamer) [inline developer-license.md:30]:
```suggestion
![store_channel_form.png](/uploads/developer-license/store_channel_form.png)
```

**Review by Rexogamer — COMMENTED** [review]
almost LGTM

**Review by sleepadru — APPROVED** [review]
image

**Pull request merged**